### ISSUE TITLE
dist.rpminspect.comparison.config mistakens /usr/include/sombok.h for a configuration file

### ISSUE THREAD

**Issue by ppisar** [rpminspect/rpminspect#222]
OSCI rpminspect dist.rpminspect.comparison.config test for sombok-2.4.0-7.el8 fails:

~~~~
                "testcase": {
                    "name": "dist.rpminspect.comparison.config",
                    "ref_url": "",
                    "test_outputs": [
                        {
                            "message": "%config file content change for /usr/include/sombok.h in sombok-devel on s390x",
                            "remedy": "Changes to %config should be done carefully.  Make sure you have installed the correct file and in the correct location.  If a package is restructuring configuration files, make sure the package can handle upgrading an existing package -or- honor the old file locations.",
                            "result": "NEEDS_INSPECTION",
                            "waiver_authorization": "Anyone"
                        },
~~~~

The file content changed from sombok-2.4.0-6.el8. But the problem is that /usr/include/sombok.h is not a %config file:

~~~~
%files devel
%defattr(-,root,root,-)
%{_includedir}/sombok*.h
%{_libdir}/libsombok.so
%{_libdir}/pkgconfig/sombok.pc
~~~~~

**Comment by dcantrell** [comment]
Definitely looks like a bug to me.  I will get working on this as soon as I can today.  Not sure what is happening here.

**Comment by dcantrell** [comment]
This bug is resolved with the fix for #221 

Changes have been pushed and new builds will show up in the Copr repo.  The next released version with this fix included will be 1.2.
